server.js

```diff
@@ -279,10 +279,20 @@ app.get("/api/db/users", async (request, response) => {
 
 app.post("/api/db/episode/:id", async (request, response) => {
   const { id } = request.params;
-  const { userId } = request.body;
+  const { userId, liked } = request.body;
   try {
-    await setEpisode(id, userId);
-    response.status(200).send("Episode is set in database.");
+    const episodeInDb = await getSingleEpisode(id);
+    if (episodeInDb) {
+      if (liked) {
+        await updateEpisodeLiked(id, userId);
+      } else {
+        await updateEpisodeUnliked(id, userId);
+      }
+      response.status(200).send("Database updated.");
+    } else {
+      await setEpisode(id, userId);
+      response.status(200).send("Episode is set in database.");
+    }
```

Comment:
Better split this into two routes, one for creation and one for updating. See HTTP method specs https://tools.ietf.org/html/rfc7231#section-4.3